File: Go/BruteForce/7568.go
package main

import (
	"fmt"
)

func main() {
	//reader := bufio.NewReader(os.Stdin)
	var tc int
	fmt.Scan(&tc)
	var h, w int
	in_list := make([][]int, 0) //2차원 slice
	for i := 0; i < tc; i++ {
		fmt.Scan(&w, &h)
		temp := make([]int, 0)
		temp = append(temp, w)
		temp = append(temp, h)
		in_list = append(in_list, temp)
	}
	result := make([]int, 0)
	for i, curr := range in_list {
		count := 1
		for j, other := range in_list {
			if i != j {
				if curr[0] < other[0] && curr[1] < other[1] {
					count++
				}
			}
		}
		result = append(result, count)
	}

	for _, val := range result {
		fmt.Printf("%d ", val)
	}

}

File: python/BruteForce/7568.py

if __name__ == "__main__":
    tc=int(input())
    in_list=list()
    for i in range(tc):
        w_h_list=list(map(int,input().split(" ")))
        in_list.append(w_h_list)
    
    result_list=list()
    fin=len(in_list)
    for i in in_list:
        count =1
        for j in in_list:
            if(i!=j):
                if(i[0]<j[0] and i[1]<j[1]):
                    count+=1
        
        result_list.append(count)
    for val in result_list:
        print(val,end=" ")